video/vaapi.c

```diff
@@ -200,6 +214,31 @@ static const struct va_native_display disp_x11 = {
 };
 #endif
 
+#if HAVE_VAAPI_WIN32
+#include <va/va_win32.h>
+
+static void win32_create(struct mpv_global *global, VADisplay **out_display,
+                         void **out_native_ctx, const char *path)
+{
+    LUID *luid = NULL;
+    DXGI_ADAPTER_DESC1 desc = {0};
+    if (path) {
```

Comment:
needs a check for empty too?

Reply:
Passing empty string is consistent with other adapter selection function, it select first adapter. But sure I can let vaapi decide.

Reply:
I was asking because I was wondering if `""` and `NULL` would then behave differently.

Reply:
There is len check down in `mp_get_dxgi_adapter`. But it is good to just not do anything in case it is empty.